common/io/SelectServerTest.cpp

```diff
@@ -582,7 +582,7 @@ void SelectServerTest::testTimeout() {
       10,
       ola::NewSingleCallback(this, &SelectServerTest::ReentrantTimeout, m_ss));
   m_ss->RegisterSingleTimeout(
-      20,
+      40,
```

Comment:
Do you mind adding some LOGs to see where the delay is. What should happen is this:

T= 0, ReentrantTimeout added, Terminate timeout added
T= 10ms, ReentrantTimeout fires, adds two timeouts
T= 10ms, first of the timeouts fires, increments the counter
T = 15ms, second of the timeouts fires, increments the counter
T = 20ms, Terminate timeout runs

If we're delaying by more than 5ms it's likely that health checked connections in E1.33 won't work correctly so it would be good to know now.